logging/cluster-logging-support.adoc

```diff
@@ -29,3 +29,30 @@ The {logging-title} is not:
 * Historical or long term log retention or storage
 * A guaranteed log sink
 * Secure storage - audit logs are not stored by default
+
+include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
+include::modules/unmanaged-operators.adoc[leveloffset=+1]
+
+[id="cluster-logging-support-must-gather"]
+== Collecting logging data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
```

Comment:
```suggestion
When opening a support case, it is helpful to provide debugging information about your cluster to Red{nbsp}Hat Support.
```
Per new SSG guidance, use a non-breaking space between `Red` and `Hat`: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces